Framework/Core/include/Framework/ASoA.h

```diff
@@ -541,8 +541,23 @@ struct Index : o2::soa::IndexColumn<Index<START, END>> {
 template <typename T>
 using is_dynamic_t = framework::is_specialization<typename T::base, DynamicColumn>;
 
+namespace persistent_type_helper
```

Comment:
Not sure I like the idea of such a specific namespace. Do you really need it?

Reply:
Then instead would you prefer the included functions to have a more specific name instead of `test`?